mettle/configure.ac

```diff
@@ -3,7 +3,7 @@ AC_INIT([mettle], m4_esyscmd([tr -d '\n' < VERSION]))
 AC_CANONICAL_HOST
 AM_INIT_AUTOMAKE([subdir-objects])
 AC_CONFIG_MACRO_DIR([m4])
-AM_MAINTAINER_MODE
+AM_MAINTAINER_MODE([enable])
```

Comment:
The idea behind this change is so you don't have to remember to reconfigure anymore: https://www.gnu.org/software/automake/manual/html_node/maintainer_002dmode.html